src/reducers/index.ts

```diff
@@ -16,7 +16,10 @@ var initialState = {
   displayOptions: {
     rootTypeId: null,
     skipRelay: true,
-    sortByAlphabet: false
+    sortByAlphabet: false,
+    hideDocs: false,
+    hideRoot: false,
+    transformSchema: null,
```

Comment:
@IvanGoncharov that is a very good point. Do you see another way to pass in the function than with redux?